frog/imports/client/Wiki/index.js

```diff
@@ -74,7 +84,8 @@ type WikiCompStateT = {
   noInstance: ?boolean,
   username: string,
   isAnonymous: boolean,
-  settings?: WikiSettingsT
+  settings?: WikiSettingsT,
```

Comment:
We should make settings not optional to help reduce the code complexity.

Reply:
Existing wikis don't have the settings object, this has been done to ensure backward compatibility

Reply:
Again, as previously mentioned, provide a default set of values for those pages to handle backward compability. The rest of the code should not be responsible for this behaviour.